app/controllers/tag_controller.rb

```diff
@@ -11,6 +11,10 @@ def index
 
     @title = I18n.t('tag_controller.tags')
     @paginated = true
+    @order_type = params[:order] == "desc" ? "asc" : "desc"
+
+    order_string = get_order_string()
```

Comment:
[nit] you don't need a `()` after a method invocation in ruby, unless you are passing in arguments. :)

Reply:
Also, I would just rename `get_order_string` and replace that function with `order_string`.

e.g. 
```
def get_order_string 
 ...
end
```

Because ruby methods are invoked simply by calling them, you can keep the `order_string` the same as below.  all `order_string` will do there is call the `order_string` function, returning the order string.  thus, there is no need to assign the value of `get_order_string` to a variable.  just call that function wherever you were using `order_string`.

Does that make sense?  It's a little bit hard to explain, but im happy to go further in detail if you need it. :)

Reply:
(and make sure to keep `order_string` a private method! :D )

Reply:
The idea of removing get_order_string and directly calling order_string is really cool :sunglasses: Thanks !!